pvcollada20_schema/pvcollada_schema_0.1.xsd

```diff
@@ -633,11 +638,13 @@
     <xs:complexType>
       <xs:sequence>
         <xs:element name="source" type="xs:string"/>
+        <!-- Why do we need this? Specifying "target software" is in conflict with PVcollada 2.0 being a common interchange format -->
```

Comment:
Why do we need this? Specifying "target software" is in conflict with PVcollada 2.0 being a common interchange format